README.md

```diff
@@ -1,19 +1,73 @@
-Liquid Prompt — a useful adaptive prompt for Bash & zsh
-=======================================================
+<h1 align="center">
+    <img
+        width="500"
+        src="https://raw.githubusercontent.com/nojhan/liquidprompt/fix(doc)-marketing/docs/liquidprompt_emblem.svg"
```

Comment:
This link needs updating now that the file has moved.

Reply:
The `docs` directory is indicated, but what it need is an update once this is merge on master.